### ISSUE TITLE
💡 Expand “Best practices” guide

### ISSUE THREAD

**Issue by dblythy** [ParsePlatform/Docs#811]
There are a number of issues that have been raised regarding firewall or other stack related issues that cannot be solved by the Parse Server package. 

I think to help our community, I propose a “best practices / recommended configuration (or even FAQ)” section. From the top of my head, I’d imagine:

-[Rate Limiting](https://github.com/parse-community/parse-server/issues/1505) - Parse wide / specific endpoints e.g /requestPasswordReset
-[Improving Performance](https://community.parseplatform.org/t/what-someone-should-do-to-increase-performance-of-parse-server/973/10)
-[Unit testing cloud code](https://github.com/parse-community/parse-server/issues/2488)

It would be good to try to get some shared knowledge going from how the core team’s configure their own servers, especially as I’d imagine most of our developers aren’t overly experienced with application stacks.

**Comment by mtrezza** [comment]
There have been threads on GitHub and/or Parse Forum regarding this. It could be helpful if you link them here to consolidate the discussion.

In addition, there have been troubleshooting threads that you could draw material from for such a best practice document.

For example:
- https://github.com/parse-community/parse-server/issues/6778

**Comment by dblythy** [comment]
Of course, I've edited the original post to include some links. If you have any suggestions for topics that could be included in the section, please feel free to post them too.

**Comment by mtrezza** [comment]
Parse Server docs do have a [Best Practice](https://docs.parseplatform.org/parse-server/guide/#best-practice) page.
